### PR TITLE
#158175308 Fix frontend test

### PR DESCRIPTION
**Pull request by ja-odur**
#### What does this PR do?
* Have front tests reviewed.
#### Description of Task to be completed?
Have the following tests working in `hc/front/test/` folder
* `test_invalid_form_entry` in `test_add_channel.py` file.
* `test_pushover_validates_priority` in `test_add_pushover.py` file.
* `test_team_access_works` in `test_add_channel.py` file.
#### How should this be manually tested?
* Clone this application's git branch and set up the application as instructed in the `README.md` file
* Open terminal and ensure you're in the application folder
* Run `./manage.py test`
#### What are the relevant pivotal tracker stories?
* [#158175308](https://www.pivotaltracker.com/story/show/158175308)

### CODE REVIEW

**Review by wasswa-derick — DISMISSED** [review]
Changes look good to me.

**Review by Yiga-fred — COMMENTED** [review]
Am looking at it  but good work

**Comment by wasswa-derick** [comment]
Joseph there seems to be a conflict with your branch's requirements.txt and the one at develop.

File: hc/front/tests/test_add_channel.py [hunk 1]
```diff
@@ -38,4 +38,26 @@ def test_instructions_work(self):
...
+        url = "/integrations/add/"
+        form = {"kind": "email", "value": "bob@example.org"}
+
+        # Logging in as bob, not alice. Bob has team access so this
+        # should work.
+        self.client.login(username="bob@example.org", password="password")
+        r = self.client.post(url, form)
+
+        self.assertRedirects(r, "/integrations/")
+        assert Channel.objects.count() == 1
```
Comment (john555) [inline hc/front/tests/test_add_channel.py]: You should use `self.assertEqual`. e.g `self.assertEqual(400, r.status_code)`. Note that it's common, and a best practice to start with the expected value followed by the returned value. https://stackoverflow.com/questions/9331259/unit-testing-why-is-the-expected-argument-always-first-in-equality-tests

File: hc/front/tests/test_add_check.py [hunk 2]
```diff
@@ -12,3 +12,12 @@ def test_it_works(self):
         assert Check.objects.count() == 1
 
     ### Test that team access works
+    def test_team_acess_works(self):
```
Comment (john555) [inline hc/front/tests/test_add_check.py]: - Make sure Bob and Alice are on the same team.
- Login as Alice and add a check
- Login as Bob and check if Bob has access to that check.

File: hc/front/tests/test_add_pushbover.py [hunk 3]
```diff
@@ -38,3 +38,15 @@ def test_it_validates_nonce(self):
...
 
     ### Test that pushover validates priority
+    def test_pushover_validates_priority(self):
+        self.client.login(username='alice@example.org', password='password')
+
+        session = self.client.session
+        session["po_nonce"] = "n"
+        session.save()
+
+        # Note that prio is set an invalid value of 3
```
Comment (john555) [inline hc/front/tests/test_add_pushbover.py]: You have a typo in your Comment ('prio')

**Review by john555 — CHANGES_REQUESTED** [review]

File: hc/front/tests/test_add_channel.py (hunk 1 above)
Comment (ja-odur) [reply hc/front/tests/test_add_channel.py]: Okay, I going change that.

File: requirements.txt [hunk 4]
```diff
@@ -5,5 +5,5 @@ django_compressor==2.1
 djmail==0.11.0
 futures==3.0.3
 premailer==2.9.6
-psycopg2==2.6.1
+psycopg2==2.7
```
Comment (john555) [inline requirements.txt]: Use `psycopg2==2.7` instead when resolving merge conflict.

**Review by john555 — CHANGES_REQUESTED** [review]

**Comment by john555** [comment]
The PR title "Fix front test" does not add up. Did you mean "Fix frontend tests"?

File: .gitignore [hunk 5]
```diff
@@ -1,7 +1,8 @@
 __pycache__/
 *.pyc
 .coverage
-.vscode/
```
Comment (john555) [inline .gitignore]: Don't remove this

**Review by john555 — CHANGES_REQUESTED** [review]
Squash your commits into one.

**Review by john555 — APPROVED** [review]

**Pull request merged**